detect_secrets/util/color.py

```diff
@@ -1,4 +1,11 @@
 from enum import Enum
+from os import getenv
+from sys import stdout
+
+
+def support_ansi_colors() -> bool:
```

Comment:
This is straight up nitpicking, but I believe the name of this function should be `supports_ansi_colors`.